config/routes.rb

```diff
@@ -6,6 +6,8 @@
   post '/validate_beta_user', to: 'static#validate_beta_user'
   get 'home', to: 'static#index'
 
+  get '/rooms/random', to: redirect { "/rooms/#{Room.random_id}" }
```

Comment:
I think this logic can go into a separate action in RoomsController

```
def random_room
  redirect_to room_path(Room.recommended.take)
end
```

Then the route can be

```
get 'rooms/random_room', to: 'rooms#random_room', as: 'random_room'
```

Reply:
I agree. don't add a custom route either, just `get "random", on: :collection` in  https://github.com/tomekr/jamroulette/blob/afdd6097d250d166fee1d3627b0e32b3c0495319/config/routes.rb#L9-L11